src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -57,9 +62,23 @@ def _check_model_args(
     """
     if not args.subcommand and not args.model:
         parser.error("The -m/--model option is required and cannot be empty.")
+    args = _convert_str_to_enum_entry(
+        args, "model_selection_strategy", ModelSelectionStrategy
+    )
+    _generate_formatted_model_name(args)
     return args
 
 
+def _generate_formatted_model_name(args: argparse.Namespace) -> None:
+    if len(args.model) == 1:
+        args.formatted_model_name = args.model[0]
+    elif len(args.model) == 0:
```

Comment:
I removed this comment. It looks like this is a workaround since we don't have a profile subcommand yet, so the testing is probably failing correctly for the compare subcommand.

Reply:
got it.
agreed.
we NEED to clean that up.